packages/odyssey-react-mui/src/PasswordField.tsx

```diff
@@ -151,7 +151,12 @@ const PasswordField = forwardRef<HTMLInputElement, PasswordFieldProps>(
             )
           }
           id={id}
-          inputProps={{ role: "textbox" }}
+          inputProps={{
+            "aria-errormessage": errorMessageElementId,
+            "aria-labelledby": labelElementId,
+            // role: "textbox" added for unit testing purposes
+            role: "textbox",
```

Comment:
The role textbox is because password inputs don’t have a role assigned.

Reply:
```suggestion
            // Added because password inputs don't have an implicit role assigned. This causes problems with element selection.
            role: "textbox",
```